nativelib/src/main/scala/scala/scalanative/runtime/Platform.scala

```diff
@@ -30,4 +30,15 @@ object Platform {
 
   @name("scalanative_wide_char_size")
   final def SizeOfWChar: CSize = extern
+
+  @name("scalanative_platform_is_msys")
```

Comment:
There is no foreign method defined with that name

Reply:
method has been added to `./nativelib/src/main/resources/scala-native/platform/platform.c`